templates/install-wp-tests.sh

```diff
@@ -147,7 +147,20 @@ install_db() {
 	fi
 
 	# create database
-	mysqladmin create $DB_NAME --user="$DB_USER" --password="$DB_PASS"$EXTRA
+	if [ $(mysql -e 'show databases;' | grep ^$DB_NAME$) ]
```

Comment:
This command needs to use the `--user="$DB_USER" --password="$DB_PASS"$EXTRA` arguments for login.

Also, is it possible that the given MySQL user would have access to _use_ the requested database, but not to _list all databases_?

Reply:
Good catch! Fixing the missing creds now.